README.md

```diff
@@ -69,24 +69,24 @@ import NativePackagerKeys._
 
 _...otherwise you will get duplicate imports reported. This is because the new 1.0+ version uses sbt's auto plugin feature._
 
-Unless the ConductR is running at http://127.0.0.1:9005, and instead supposing that it is running at
-http://192.168.59.103:9005 then you will typically issue the following command from the sbt console:
+Unless the ConductR is running at `127.0.0.1:9005`, and instead supposing that it is running at
+`192.168.59.103:9005` then you will typically issue the following command from the sbt console:
 
 ```
-conductr http://192.168.59.103:9005
+conductr:controlServer 192.168.59.103:9005
```

Comment:
We can abolish this command as a result of now having `CONDUCTR_IP` and `CONDUCTR_PORT`.